testSrc/unit/io/flutter/editor/ActiveEditorsOutlineServiceTest.java

```diff
@@ -39,11 +39,11 @@
   // The outline service doesn't care too much what the outline's contents are, only that the outlines update.
   // However, the request method uses the outline's length to determine if the outline is up-to-date.
   private static final FlutterOutline firstFlutterOutline =
-    new FlutterOutline("First", 0, 0, 0, 0, "", null, null, null, null, null, null, 1, false, null, null, null, null);
+    new FlutterOutline("First", 0, 0, 0, 0, "", null, null, null, null, null, null);
```

Comment:
update the test to match the latest version of the FlutterOutline class. The existing version was about a year out of date.